bamboo/bootstrap-tf-deployment.sh

```diff
@@ -41,12 +41,6 @@ echo "Deploying Cumulus data-persistence module to $DEPLOYMENT"
   -var "aws_region=$AWS_REGION" \
   -var "subnet_ids=[\"$AWS_SUBNET\"]"
 
-# Test that deployment succeeded by failing on bad exit code.
```

Comment:
It's redundant - the script is set `-e` and should fail for any exit code that isn't 0 anyway, it's unclear to me why this was ever there, unless the intent was to avoid `-e`, and even that doesn't make sense in this context.